app/views/shared/_claim_history.html.haml

```diff
@@ -1,21 +1,22 @@
-- if current_user.persona.is_a?(CaseWorker)
-  - open_messages_url = messages_case_workers_claim_path(claim)
-  - message_controls_url = show_message_controls_case_workers_claim_path(claim)
-- else
-  - open_messages_url = messages_external_users_claim_path(claim)
-  - message_controls_url = show_message_controls_external_users_claim_path(claim)
+#claim-accordion
```

Comment:
This partial is used elsewhere (in `app/views/shared/_claim_accordion.html.hml`, which is still used for external users) so this change will affect that. Does the `claim-accordion` id need to be here? Could it be used in the `_claim_status.html.haml`, where this partial is used?